README.md

```diff
@@ -35,6 +36,14 @@ The package has been developed and tested with:
 4. Create a `log` folder in the root of your site. The import script will need to write report files to this folder, you may need to set the permissions on the folder.
 5. Add `"wagtail_wordpress_import"` to your INSTALLED_APPS config in your settings.py file.
 
+### BASE_URL for importing images and documents
+
+The importer will need to know what the URL is for the website to download images and documents from.
+
+Set the `BASE_URL` setting in your sites settings to the URL of the website that the images and documents will be imported from.
+
+*If you need to use a different URL in `BASE_URL` you can use `WAGTAIL_WORDPRESS_IMPORTER_BASE_URL`.*
```

Comment:
**Comment**: I'd say this is the other way round — `WAGTAIL_WORDPRESS_IMPORTER_BASE_URL` is the standard way of setting it, but if not set it defaults to BASE_URL. On the other hand, I'm not sure why it defaults to another setting. Some projects may make use of it, but it's not part of core Django.

1. Would it make more sense to remove the fallback?
1. Would it make more sense to implementers if we renamed this to e.g. "WAGTAIL_WORDPRESS_IMPORTER_SOURCE_DOMAIN" (or something else, if I've misunderstood the role)?

As far as I see, this is only used in test settings overrides, and in one configuration module.

Reply:
Thanks Nick, what you are saying does make sense. I will alter the implementation for this and update the docs.